pyjsonq/query.py

```diff
@@ -441,3 +441,27 @@ def sort_by(self, property, order="asc"):
                 )
 
         return self
+
+    def chunk(self, size = 0):
+
+        """Group the resulted collection to multiple chunk
+
+        :@param size: 0
+        :@type size: integer
+
+        :@return Chunked List
+        """
+
+        if size == 0:
+            return ValueError('Invalid chunk size')
+
+        self.__prepare()
+        _newContent = []
```

Comment:
change the variable name `_newContent` to `_chunk` or `_new_content`. because it's not a pythonetic way.